config/overrides.go

```diff
@@ -34,6 +34,13 @@ func IdentifierAssignedByEquinix() upconfig.ResourceOption {
 	}
 }
 
+// LongProvision will set the resource to be provisioned asynchronously. Use this for resources with >1m provisions
+func LongProvision() upconfig.ResourceOption {
+	return func(r *upconfig.Resource) {
+		r.UseAsync = true
```

Comment:
This may be a premature optimization. I haven't seen any problem with deploying devices thus far.